src/lib/viewers/media/DashViewer.js

```diff
@@ -219,7 +219,7 @@ class DashViewer extends VideoBaseViewer {
      * @param {boolean} [adapt] - Enable or disable adaptation
      * @return {void}
      */
-    enableAdaptation(adapt = true) {
+    enableAdaptation(adapt) {
```

Comment:
Is `adapt` a required param, now? If so, update doc above to remove square braces